apps/src/componentLibrary/dropdown/simpleDropdown/simpleDropdown.module.scss

```diff
@@ -37,13 +45,19 @@
     position: relative;
 
     &:after {
-      content: "\f078";
-      font-family: "Font Awesome 6 Pro", sans-serif;
+      content: '\f078';
```

Comment:
Same question about quotes

Reply:
I can save without formatting but it seems intentional to prefer single quotes 🤷